glotaran/builtin/models/kinetic_image/test/test_irf.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import xarray as xr
+
+from glotaran.parameter import ParameterGroup
+from glotaran.builtin.models.kinetic_image import KineticImageModel
+
+
+def test_measured_irf():
+    model = KineticImageModel.from_dict({
+        'initial_concentration': {
+            'j1': {
+                'compartments': ['s1'],
+                'parameters': ['2']
+            },
+        },
+        'megacomplex': {
+            'mc1': {'k_matrix': ['k1']},
+        },
+        'k_matrix': {
+            "k1": {'matrix': {("s1", "s1"): '1', }}
+        },
+        'irf': {
+            'irf1': {'type': 'measured', 'method': 'conv1'},
+        },
+        'dataset': {
+            'dataset1': {
+                'initial_concentration': 'j1',
+                'irf': 'irf1',
+                'megacomplex': ['mc1'],
+            },
+        },
+    })
+
+    initial = ParameterGroup.from_list([101e-4, [1, {'vary': False, 'non-negative': False}]])
+    wanted = ParameterGroup.from_list([101e-3, [1, {'vary': False, 'non-negative': False}]])
+
+    time = np.asarray(np.arange(-10, 50, 1.5))
+    axis = {"time": time, "pixel": np.asarray([0])}
+
+    center = 0
+    width = 5
+    irf = (1/np.sqrt(2 * np.pi)) * np.exp(-(time-center) * (time-center)
+                                          / (2 * width * width))
+    irf = xr.DataArray(irf, coords=[('time', time)])
+    clp = xr.DataArray([[1]], coords=[('pixel', [0]), ('clp_label', ['s1'])])
+
+    print(model.validate())
+    assert model.valid()
+
+    print(model.validate(wanted))
+    print(wanted)
+    assert model.valid(wanted)
+
+    print(model.validate(initial))
+    assert model.valid(initial)
+
+    print(model.markdown(initial))
+
+    extra = {'irf1': irf}
+
+    dataset = model.simulate('dataset1', wanted, axis, clp, extra=extra)
```

Comment:
Could you also test a normal parameterized gaussian irf, but then fitted with a measured irf? 
This seems to only simulate with a measured_irf and then also fit with measured_irf.

Reply:
This test is only basic and to be extended after we accepted the convolution functions.